.travis.yml

```diff
@@ -1,19 +1,42 @@
 sudo: required
 dist: trusty
```

Comment:
Work on trusty
https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming?utm_source=web&utm_medium=banner&&utm_campaign=trusty-default